features/step_definitions/common_steps.rb

```diff
@@ -346,7 +346,7 @@
 end
 
 When /I click the summary (table|list) #{MAYBE_VAR} (link|button) for #{MAYBE_VAR}$/ do |table_or_list, link_name, elem_type, field_to_edit|
-  row = page.find(table_or_list == "table" ? "td" : "dt", exact_text: field_to_edit).ancestor(table_or_list == "table" ? "tr" : "dl > *")
+  row = page.find(:xpath, "//*/#{table_or_list == "table" ? "td" : "dt"}[normalize-space(.) = '#{field_to_edit}']").ancestor(table_or_list == "table" ? "tr" : "dl > *")
```

Comment:
We could do string interpolation using `#{variable}` syntax i.e. http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html

Reply:
Thanks, I've just updated to do that